ocs_ci/helpers/cephfs_stress_helpers.py

```
@@ -1015,3 +1038,345 @@ def collect_stress_job_pod_logs(stress_job_obj, dir_name=None):
+                                )
+
+                                hang_info = json.loads(marker_content)
+                                hang_info["pod_name"] = pod_name
+                                hang_markers_found.append(hang_info)
+
+                                logger.error(
+                                    f"Filesystem hang detected in pod {pod_name}:\n"
+                                    f"  Monitor Type: {hang_info.get('monitor_type')}\n"
+                                    f"  Command: {hang_info.get('command')}\n"
+                                    f"  Timestamp: {hang_info.get('timestamp')}\n"
+                                    f"  Details: {hang_info.get('details')}"
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)